### DISCUSSION TITLE
I need help with search in page function

### DISCUSSION THREAD

**Discussion by RomanAleynikov** [hoothin/SearchJumper#124] [Q&A] [answered]
Hi!
First of all I want to thank you for this wonderful extension! I searched extension with searching on page function and highlighting function very long time. Finally I found it some days ago! It is just perfect! There are some simular extensions in chrome store but most of them are not free. But this extension includes all features of that not free extensions. It is very nice. It is not often that. 

I wrote before I use only "find on page" function and I have one question. What do I need to do to search each word of entered sentence differently? I send an image in which there is  I mean.

Thank you so much!
![2024-05-12_031408](https://github.com/hoothin/SearchJumper/assets/87813216/f93ab843-a749-4866-8f26-73434d2d5ab3)

**Comment by hoothin** [comment, marked as answer]
![image](https://github.com/hoothin/SearchJumper/assets/5361213/9b44bd65-b132-48e8-894c-fff09da169dc)
Click this button.

**Reply by RomanAleynikov** [reply]
Thank you!
